eventstore/migrations/0051_auto_20211213_1038.py

```diff
@@ -0,0 +1,50 @@
+# Generated by Django 2.2.24 on 2021-12-13 10:38
+
+import uuid
+
+import django.contrib.postgres.fields.jsonb
+import django.utils.timezone
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("eventstore", "0050_askfeedback")]
+
+    operations = [
+        migrations.CreateModel(
+            name="Feedback",
+            fields=[
+                (
+                    "id",
+                    models.UUIDField(
+                        default=uuid.uuid4,
+                        editable=False,
+                        primary_key=True,
+                        serialize=False,
+                    ),
+                ),
+                ("contact_id", models.UUIDField()),
+                (
+                    "functionality",
+                    models.CharField(
+                        choices=[("ask", "Ask"), ("profile_update", "Profile Update")],
+                        max_length=20,
+                    ),
+                ),
+                ("positive", models.BooleanField(default=False)),
+                ("timestamp", models.DateTimeField(default=django.utils.timezone.now)),
+                (
+                    "created_by",
+                    models.CharField(blank=True, default="", max_length=255),
+                ),
+                (
+                    "data",
+                    django.contrib.postgres.fields.jsonb.JSONField(
+                        blank=True, default=dict, null=True
+                    ),
+                ),
+            ],
+        ),
+        migrations.DeleteModel(name="AskFeedback"),
```

Comment:
This migrate is doing a create model + delete model. Just want to make sure that this isn't something we have data in, otherwise we're going to lose all that data instead of migrating it to the new model.

Reply:
Yeah its not in prod yet so all good, thanks for reviewing!